R/about.R

```diff
@@ -0,0 +1,16 @@
+aboutTab <- function() {
+  shiny::div(
+  class = "about",
+  shiny::tags$h2(
+    tags$span("Shiny App to review PhenotypeR results"))  ,
+  shiny::tags$h4( tags$span("Here more information about the use of the app ")),
+  img(
+    src = here::here("extras/www/images/hds_logo.svg"),
```

Comment:
if you save the file in inst folder then you can access it doing:
system.file("my_file.csv", package = "omopViewer")